table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveLayout.java

```diff
@@ -100,14 +100,27 @@ public PartitionInfo getData() {
     return mPartitionStatsInfo;
   }
 
+  @Override
+  public String getLocation() {
+    return mPartitionInfo.getStorage().getLocation();
+  }
+
+  private HiveLayout transformLayout(AlluxioURI transformOutputPath) {
+    PartitionInfo info = mPartitionInfo.toBuilder()
+        .setStorage(mPartitionInfo.getStorage().toBuilder()
+            .setLocation(transformOutputPath.getPath())
```

Comment:
should we be using `transformOutputPath.toString()`  to get the  scheme and authority as well?

Reply:
yes, I now put scheme and authority into the location too